plugins/plugins.go

```diff
@@ -759,6 +761,23 @@ func (m *Manager) ConsoleLogger() logging.Logger {
 	return m.consoleLogger
 }
 
+// ServerInitialized signals a channel indicating that the OPA
+// server has finished initialization.
+func (m *Manager) ServerInitialized() {
+	select {
+	case m.serverInitialized <- struct{}{}:
```

Comment:
I think closing the channel could be preferable: multiple waiters could be signalled that way. Sending a thing would mean only one of multiple readers would get the message.

But I can double check this later, not at a computer right now.

Reply:
Ah good call, makes sense, I'm still a golang noob 😁